Bottom: cfd227c7e119fe002974cde3b81658553caeba61
Top:    fe857cb32e584e64908991a3c8f65184c10acff4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2011-12-25 00:47:28 +0000

Refresh of hid-gtk-avoid-warnings-when-ne

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index f8a02ed..35e39b5 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -342,8 +342,6 @@ static void draw_cap (Coord width, int x, int y, double angle)
 {
   float radius = width / 2.;
 
-return;
-
   CHECK_IS_IN_CONTEXT ();
 
   hidgl_ensure_vertex_space (&buffer, 6);
@@ -397,7 +395,7 @@ hidgl_draw_line (int cap, Coord width, Coord x1, Coord y1, Coord x2, Coord y2, d
   switch (cap) {
     case Trace_Cap:
     case Round_Cap:
-      circular_caps = 0;
+      circular_caps = 1;
       break;
 
     case Square_Cap:
